WordPress/openverse-frontend
# Add migration notice and translation banner to the blank layout; fix translation banner logic

## Fixes


Fixes #784 by @zackkrida

## Description


This PR:
- adds the migration notice and translation banner to the blank layout (used on the homepage and 404)
- changes the logic to detect whether we should show the translation banner. Since we moved the locale detection to path-based, we remove the WordPress theme - Nuxt locale passing messages, and use the app i18n locale to detect whether we should show the banner.
- adds the simple e2e tests for the translation banner.

## Testing Instructions

Run the app and go to `http://localhost:8443/ru/`. You should be able to see the translation banner at the top. From there, you can go to the Russian Openverse translation page, or close the banner. If you navigate to other pages, you should not see the banner after closing it once. Also, you shouldn't see the banner at `http://localhost:8443`.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
